tests/api/test_get_resource.py

```diff
@@ -0,0 +1,486 @@
+    })[0]
+    task = KojiTask.get_or_create({
+        'id_': '2701',
+        'method': 'some_method',
+        'priority': 2,
+        'start_time': datetime(2017, 4, 2, 19, 39, 6),
+        'create_time': datetime(2017, 4, 2, 19, 30, 6),
+        'completion_time': datetime(2017, 4, 2, 19, 45, 6),
+        'state': 3,
+        'weight': 123.45,
+        'arch': 'no_arch'
+    })[0]
+    tag = KojiTag.get_or_create({
+        'id_': '2702',
+        'name': 'some_active_tag'
+    })[0]
+    advisory = Advisory.get_or_create({
+        'actual_ship_date': datetime(2017, 8, 1, 15, 43, 51),
+        'advisory_name': 'RHBA-2017:2251-02',
+        'content_types': ['docker'],
+        'created_at': datetime(2017, 4, 3, 14, 47, 23),
+        'id_': '27825',
+        'issue_date': datetime(2017, 8, 1, 5, 59, 34),
+        'product_name': 'Red Hat Enterprise Linux',
+        'product_short_name': 'RHEL',
+        'security_impact': 'None',
+        'state': 'SHIPPED_LIVE',
+        'status_time': datetime(2017, 8, 1, 15, 43, 51),
+        'synopsis': 'cifs-utils bug fix update',
+        'type_': 'RHBA',
+        'update_date': datetime(2017, 8, 1, 7, 16),
+        'updated_at': datetime(2017, 8, 1, 15, 43, 51)
+    })[0]
+    adv_state = AdvisoryState.get_or_create({
+        'id_': '7890',
+        'created_at': datetime(2017, 4, 2, 19, 45, 6),
+        'updated_at': datetime(2017, 4, 2, 19, 45, 6),
+        'name': 'some_name'
+    })[0]
+    fm_event = FreshmakerEvent.get_or_create({
+        "event_type_id": 8,
+        "id_": "1180",
+        "message_id": "ID:messaging-devops-broker01.test",
+        "state": 2,
+        "state_name": "COMPLETE",
+        "state_reason": "All container images have been rebuilt",
+        "url": "/api/1/events/1180"
+    })[0]
+    cb = ContainerBuilds.get_or_create({
+        "build_id": 15639047,
+        "event_id": 1180,
+        "id_": "397",
+        "name": "jboss-eap-7-eap70-openshift-docker",
+        "original_nvr": "jboss-eap-7-eap70-openshift-test",
+        "rebuilt_nvr": "jboss-eap-7-eap70-openshift-test",
+        "state": 1,
+        "state_name": "DONE",
+        "state_reason": "Built successfully.",
+        "time_completed": datetime(2017, 4, 2, 19, 39, 6),
+        "time_submitted": datetime(2017, 4, 2, 19, 39, 6),
+        "type_": 1,
+        "type_name": "IMAGE",
+        "url": "/api/1/builds/397"
+    })[0]
+
+    if resource == 'bugzillabug':
+        bug.assignee.connect(mprahl)
+        bug.qa_contact.connect(jsmith)
+        bug.reporter.connect(tbrady)
+        commit.resolved_bugs.connect(bug)
+        commit_two.resolved_bugs.connect(bug)
+        commit_three.reverted_bugs.connect(bug)
+        advisory.attached_bugs.connect(bug)
+
+    if resource == 'distgitcommit':
+        commit.author.connect(tbrady)
+        commit.committer.connect(jsmith)
+        commit.parent.connect(commit_two)
+        commit_three.parent.connect(commit)
+        commit.related_bugs.connect(bug)
```

Comment:
The relationships are all with the same bug. Can you create other ones to test that the relationships are correct?